ddtrace/tracer/tracer_test.go

```diff
@@ -1095,6 +1098,24 @@ func decode(p *payload) (spanLists, error) {
 	return traces, err
 }
 
+func (t *dummyTransport) waitFlush(n int, timeout time.Duration) {
+	a := time.After(timeout)
```

Comment:
What I've generally done and seems a bit nicer to read in the `select` case is:
```
timeout := time.After(5*time.Second)
```
I don't think you have to make the timeout duration a parameter. It's not really changing based on various calls.

Reply:
This needs to be done because otherwise a new timeout is started each time through the loop, and the default branch is always taken. We want the timeout to apply over all the iterations, not over an individual select, so the timeout needs to be started before the loop starts.

You're right about the timeout. We can hard-code it since it's not used.